content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+The Private Cloud Gateway (PCG) enables support for isolated private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster. 
+<br />
+
+<InfoBox>
+
+For production environments, we recommend using three nodes. If you initially set up the gateway with one node, you can resize it at a later time. 
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a docker container that is temporarily deployed on your laptop, workstation or jump-box. You can run the installer on any system that has a docker daemon installed and connectivity to Palette and the MAAS identity endpoint.
+
+
+# Prerequisites
+
+- A linux computer with a docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system.
+
+- One IP address for a Kubernetes control-plane.
+
+- One IP address for a single-node gateway or three IP addresses for a three-node gateway.
+
+- Sufficient IPs for application workload services, such as Load Balancer services.
+
+<WarningBox>
+
+By default, the MAAS Kubernetes pack uses 192.168.0.0/16. Ensure that the Pod CIDR range for any clusters you deploy after setting up the PCG do not overlap with the network used by the bare metal machines that MAAS manages.
+
+</WarningBox>
+
+- Minimum capacity requirements as follows:
+
+    - One node (no HA): 4 vCPU, 8 GiB Memory, 60 GiB Storage.
+    - Three nodes (HA): 6 vCPU, 12 GiB Memory, 90 GiB Storage.<br /><br />
+
+- An active MAAS API key in the format  ``[consumer_key], [key], and [secret] tokens: API key = '[consumer_key]:[key]:[secret]'``.
+
```

Comment:
```suggestion
- An active MAAS API key in the format  ``[consumer_key], [key], and [secret] tokens: API key = '[consumer_key]:[key]:[secret]'``. Refer to the [MAAS API authentication](https://maas.io/docs/api-authentication-reference) to learn more.

```
This can be confusing for new users so let's reference the maas API auth docs.

Reply:
done